### ISSUE TITLE
Search bar issues

### ISSUE THREAD

**Issue by bwhicks** [Princeton-CDH/derrida-django#207]
- [ ] Fix filter being skipped in tab navigation (probably because selectize.js doesn't have aria roles and doesn't use default form functionality)
- [ ] Resolve orphan tag issues (probably by giving them approriate `for=` attributes if possible)

**Comment by rlskoeser** [comment]
Made some small improvements but I'm not sure how to fix the select. I think we may only be using selectize so we can style the select and not any additional functionality; in which case, we might want to switch to something simpler and/or designed for accessibility. Maybe we could use something like this [deque select](https://pattern-library.dequelabs.com/components/selects).

**Comment by kmcelwee** [comment]
Archiving site, making this unnecessary
